services/controllerhost/queueDepth.go

```diff
@@ -329,9 +326,13 @@ func (qdc *queueDepthCalculator) computeBacklog(cgDesc *shared.ConsumerGroupDesc
 
 	switch qdc.iter.isDLQ {
 	case true:
-		backlog = common.MaxInt64(0, storeMetadata.lastSequence-(storeMetadata.beginSequence+1))
+		if storeMetadata.lastSequence != math.MaxInt64 && storeMetadata.beginSequence != math.MaxInt64 {
+			backlog = storeMetadata.lastSequence - storeMetadata.beginSequence + 1
+		}
 	case false:
-		backlog = common.MaxInt64(0, storeMetadata.availableSequence-cgExtent.GetAckLevelSeqNo())
+		if storeMetadata.availableSequence != math.MaxInt64 {
+			backlog = storeMetadata.availableSequence - cgExtent.GetAckLevelSeqNo()
```

Comment:
you want to still leave the common.MaxInt64(0, backlog) as is - these values are eventually consistent.. So, there is no strong guarantee that AckLevelSeqNo will be less than the availableSequence. And to be consistent, you can do the same thing for if (true) case